lmdeploy/pytorch/chat.py

```diff
@@ -132,16 +132,16 @@ def run_chat(model_path,
             nth_round += 1
 
 
-def main(model_path,
+def main(model_path: str,
          model_name: str = None,
          session_id: int = 1,
-         top_k=40,
-         top_p=0.8,
-         temperature=0.8,
+         top_k: float = 40,
+         top_p: float = 0.8,
+         temperature: float = 0.8,
          repetition_penalty: float = 1.0,
          tp: int = 1,
          stream_output: bool = True,
-         adapter: str = None,
+         adapter: Dict[str, str] = None,
```

Comment:
This is designed to load only one adapter.
What will happen if we start this in fire?

Reply:
for python fire, it accept python str of dict: `--adapter '{k:v}'`